pkg/ragengine/controllers/ragengine_controller_test.go

```diff
@@ -196,23 +197,23 @@ func TestCreateAndValidateMachineNodeforRAGEngine(t *testing.T) {
 	testcases := map[string]struct {
 		callMocks        func(c *test.MockClient)
 		cloudProvider    string
-		objectConditions apis.Conditions
+		objectConditions []status.Condition
```

Comment:
Is the status imported from "github.com/awslabs/operatorpkg/status" ? why not from karpenterv1?